robolectric-shadows/shadows-core/src/main/resources/org/robolectric/shadows/ShadowAccessibilityNodeInfo.java.vm

```diff
@@ -545,20 +891,36 @@ public class ShadowAccessibilityNodeInfo {
    */
   @Implementation
   public int getActions() {
-    int actions = 0;
-    actions |= (isClickable()) ? AccessibilityNodeInfo.ACTION_CLICK : 0;
-    actions |= (isLongClickable()) ? AccessibilityNodeInfo.ACTION_LONG_CLICK : 0;
-    actions |= (isScrollable()) ? AccessibilityNodeInfo.ACTION_SCROLL_FORWARD : 0;
-    actions |= (isScrollable()) ? AccessibilityNodeInfo.ACTION_SCROLL_BACKWARD : 0;
-#if ($api >= 18)
-    actions |= (isPasteable()) ? AccessibilityNodeInfo.ACTION_PASTE : 0;
-    actions |= (isTextSelectionSetable()) ? AccessibilityNodeInfo.ACTION_SET_SELECTION : 0;
+#if ($api >= 21)
+    int returnValue = 0;
+    if (actionsArray == null) {
+      return returnValue;
+    }
+
+    final int actionSize = actionsArray.size();
+    for (int i = 0; i < actionSize; i++) {
+      int actionId = actionsArray.get(i).getId();
+        if (actionId <= (int)ReflectionHelpers.getStaticField(AccessibilityNodeInfo.class, "LAST_LEGACY_STANDARD_ACTION")) {
+          returnValue |= actionId;
```

Comment:
This is probably worth a comment, "Custom actions are only returned by getActionsList().